### ISSUE TITLE
Fomod installer option preview not updated when options are choosen

### ISSUE THREAD

**Issue by lampuiho** [TanninOne/modorganizer#569]
Choosing different options does not show the preview picture accordingly. Only the default one is shown.

**Comment by ThosRTanner** [comment]
It doesn't show the picture when you chose an option. It shows you the picture as you mouse over the option.

**Comment by lampuiho** [comment]
No, just tried it again. Doesn't show me a picture, only tooltip about it being an optional component.

**Comment by TanninOne** [comment]
There is a known problem that certain windows themes prevent the hover-event that the fomod installer uses from being fired.
I don't think this can be fixed on the MO-side without changing functionality to click-to-preview.
